ios/FluentUI/Card/CardView.swift

```diff
@@ -409,19 +418,19 @@ open class CardView: UIView {
     private func setupColors() {
         switch colorStyle {
         case .appColor:
```

Comment:
Looks like we lost the variation that had the brand colors. Can you follow up with design, and if that was intentional, create a work item to track the removal of the unused styles? Or at least to condense the shared logic. I guess there's still a variation that uses brand color for the icon.

Reply:
The brand colors were updated on the last commit. Design omitted them on the Figma but they gave me the right tokens this morning.